src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -339,6 +353,12 @@ function commandHasRequiredField(command) {
  * "Enum" bits on the enum names while we're here.
  */
 function objCEnumName(clusterName, enumLabel, options) {
+  if (clusterName == 'Globals') {
```

Comment:
We should probably start defining these loose hanging strings in db-enum for better maintenance in the future.

Reply:
Possible.  What should _really_ happen is that this helper should not be in ZAP at all, but in the SDK.  ;)